x-pack/metricbeat/module/cloudfoundry/_meta/fields.yml

```diff
@@ -8,6 +8,10 @@
       type: group
       description: >
       fields:
+        - name: type
+          type: keyword
+          description: >
+            The type of event from Cloud Foundry.
```

Comment:
Nit. Add some possible values as example to the description.

Reply:
Added.